### PR TITLE
chore(deps): update javascript dependencies

### LINKED ISSUE #53: 🐞 [Bug Report] - NPM Package deprecation
**Describe the bug**

```

npm WARN deprecated rollup-plugin-terser@7.0.2: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-terser
npm WARN deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
```

**To Reproduce**
1. Run `npm i`
2. Get error

**Expected behavior**

No error

@pattishin plz advise

### PR DESCRIPTION
**Pull request by glasnt**
## Description

Fixes #53
Fixes #363 
Fixes #370

Uses `npx npm-check-updates` to update packages, plus some manual updates for compatibility, linting, etc. 

Ephemeral testing instance: [avocano-ephemeral-ad80aef1](https://avocano-ephemeral-ad80aef1.web.app/)

**Note:** Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] Added steps to reproduce the changes in this pull request
- [ ] Added relevant testing in this pull request
- [ ] Please **merge** this PR for me once it is approved.

### CODE REVIEW

**Review by iennae — APPROVED** [review]
LGTM

File: client/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "format": "eslint --ext .js . --fix --ignore-path .gitignore && prettier \"**/*.js\" --write --ignore-path .gitignore",
     "test": "web-test-runner 'test/**/*.test.js' --node-resolve",
     "test:watch": "web-test-runner --watch",
-    "build": "rimraf dist && rollup -c rollup.config.js",
+    "build": "rimraf dist && rollup -c rollup.config.mjs",
```
Comment (pattishin) [inline client/package.json:12]: Nice 👍🏼

File: client/src/utils/fetch.js [hunk 2]
```diff
@@ -94,8 +94,6 @@ export const buyProduct = async (productId, callback) => {
   } catch (error) {
     console.error(error);
   }
-
-  return { data };
```
Comment (pattishin) [inline client/src/utils/fetch.js:98]: This makes sense since we're no longer using the return.

**Review by pattishin — APPROVED** [review]
:shipit: lgtm

**Pull request merged**